source/boards/groups-filter-sort.rst

```diff
@@ -34,11 +34,11 @@ To ungroup cards on table view, select the **Group by** option at the top of the
 Filters
 -------
 
-You can filter cards on your board if they utilize the **Select** property.
+You can filter cards on your board using all the text properties.
```

Comment:
@wuwinson, is this accurate and should I also mention Filters now support all text properties! You’re no longer limited to only using filters with the Select property type. You can filter your board more granularly to see the cards most relevant to you. Boards now also supports filtering by the following property types:
- Text
- Email
- Phone
- URL

Reply:
Hey Justine, users can still filter by the *Select* property as well, so many some tweaks here.